pkgs/by-name/dr/dragmap/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  boost,
+  cmake,
+  gtest,
+  zlib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "DRAGMAP";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "Illumina";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21